lochness/redcap/data_trigger_capture.py

```diff
@@ -130,18 +129,6 @@ def get_info_from_post_body(var_name, body):
     return re.search(f'{var_name}={pattern_catcher}', body).group(1)
 
 
-def get_sha(file_loc: str) -> str:
-    '''return sha256 hexdigest of a file'''
-    BUF_SIZE = 65536
-    sha256 = hashlib.sha256()
-
-    with open(file_loc, 'rb') as f:
-        data = f.read(BUF_SIZE)
-        sha256.update(data)
-
-    return sha256.hexdigest()
```

Comment:
Did this function exist from pre-historic stage in Lochness?

Reply:
This is the code for data entry trigger for REDCap. I just moved checksum function to `lochness.utils.checksum`